src/index.tsx

```diff
@@ -25,11 +25,9 @@ export function useMappedState<TState, TResult>(
   const [mappedState, setMappedState] = useState(() => runMapState());
 
   // If the store or mapState change, rerun mapState
-  const [prevStore, setPrevStore] = useState(store);
-  const [prevMapState, setPrevMapState] = useState(() => mapState);
+  const prevStore = usePrevious(store);
+  const prevMapState = usePrevious(mapState);
```

Comment:
So these two don't work correctly. I'm not exactly sure why we need `useState` here. But since we do, it isn't really worth introducing `usePrevious` as a helper.